Makefile.in

```diff
@@ -429,7 +429,7 @@ CPPCHECK = src/cppcheck_filtered
 CPPCHECK_FLAGS = --enable=all --inline-suppr \
 	--suppress=unusedScopedObject --suppress=cstyleCast --suppress=useInitializationList \
 	--suppress=nullPointerRedundantCheck
-CPPCHECK_FLAGS += --xml
+CPPCHECK_FLAGS += --xml --std=c++03
```

Comment:
Unfortunately I think that will break checking the code that does need c++11 for threading.  We've got only another few months dealing with pre-C11 so suggest we just live with this hole.